test/Chainweb/Test/CutDB.hs

```diff
@@ -268,7 +269,7 @@ startTestPayload
     -> ChainwebVersion
     -> LogFunction
     -> Int
-    -> IO (Async (), Async(), CutDb RocksDbCas, PayloadDb RocksDbCas)
+    -> IO (Async (), Async(), CutDb RocksDbCas)
```

Comment:
The `PayloadDb` is redundant here, since it is already contained in the `CutDb` and can be accessed via `cutDbPayloadCas`.